src/main/thrift/parquet.thrift

```diff
@@ -211,6 +206,97 @@ struct Statistics {
    4: optional i64 distinct_count;
 }
 
+/** Empty structs to use as logical type annotations */
+struct StringType {}
+struct MapType {}
+struct ListType {}
+struct EnumType {}
+struct DateType {}
+
+/**
+ * Logical type to annotate a column that is always null.
+ *
+ * Sometimes when discovering the schema of existing data values are always
+ * null and the physical type is assumed.
+ */
+struct NullType {}
+
+/**
+ * Decimal logical type annotation
+ *
+ * To maintain forward-compatibility in v1, implementations using this logical
+ * type must also set scale and precision on the annotated SchemaElement.
+ */
+struct DecimalType {
+  1: required i32 scale
+  2: required i32 precision
+}
+
+/** Time units for logical types */
+struct MilliSeconds {}
+struct MicroSeconds {}
+union TimeUnit {
+  1: MilliSeconds MILLIS
+  2: MicroSeconds MICROS
+}
+
+/** Timestamp logical type annotation */
+struct TimestampType {
+  1: required bool isAdjustedToUTC
```

Comment:
if isAdjusted to UTC is true, we need to record the original TZ. So we should add an optional String timezone field. (and then we don't need isAdjustedToUTC since it is implied by timezone != null )
Something similar similar to Arrow:
https://github.com/apache/arrow/blob/a4f29f3a3ff1c64a6f547bfb0d5e4500142ea5ec/format/Schema.fbs#L117